packages/devtools/lib/src/timeline/timeline_protocol.dart

```diff
@@ -532,6 +533,22 @@ class TimelineEvent {
     }
   }
 
+  /// Returns the depth of this TimelineEvent tree, including [this].
+  int getDepth() {
+    int maxDepth(TimelineEvent root) {
```

Comment:
nit: rename maxDepth to getDepth and make it an instance method of the 
TimelineEvent object.

Reply:
done.